_posts/2022-09-07-#25957.md

```diff
@@ -10,14 +10,26 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- This PR is a re-attempt of [PR 15845](https://github.com/bitcoin/bitcoin/pull/15845)
+from 2019, which was closed without being merged. PR 15845 was the
+subject of an earlier [review club](https://bitcoincore.reviews/15845).
+- Please see the notes for that review club meeting, as they apply here too.
```

Comment:
Could use the notes here to highlight the differences between 15845 and 25957?

Reply:
+1, at least describe descriptors